RallyHereIntegration/Source/RallyHereIntegration/Private/RH_LocalPlayerSessionSubsystem.cpp

```diff
@@ -320,18 +320,29 @@ URH_SessionView* URH_LocalPlayerSessionSubsystem::CreateOrUpdateRHSession(const
 
 		if (ExistingRHSession != nullptr)
 		{
-			SCOPED_NAMED_EVENT(RallyHere_BroadcastSessionUpdated, FColor::Purple);
-			BLUEPRINT_OnSessionUpdatedDelegate.Broadcast(RHSession);
-			OnSessionUpdatedDelegate.Broadcast(RHSession);
+			// fire updated callbacks
```

Comment:
You could just fire this callback outside of the `if` since it happens regardless, and then wouldn't need to duplicate it (assuming the order of the `SessionAdded` and `SessionUpdated` doesn't matter)

Reply:
I just saw your comment in the thread about explicitly doing it after